### DISCUSSION TITLE
Windows Container

### DISCUSSION THREAD

**Discussion by arabzade** [open-telemetry/opentelemetry-collector#6308] [Q&A]
How can I run otel collector on a windows container. when I try otel/opentelemetry-collector:0.60.0, I get the error I mentioned below. I tried with the latest release as well and am still getting the same error.

docker: no matching manifest for windows/amd64 10.0.22621 in the manifest list entries.

**Comment by anandmahurakar** [comment]
I m also facing same error, Anybody has solution?

**Comment by atoulme** [comment]
You can build your own docker image for Windows or open an issue asking that the project delivers a Windows specific docker image.
